src/query/RelayQuery.js

```diff
@@ -214,6 +218,36 @@ class RelayQueryNode {
     return children;
   }
 
+  shouldBeIncluded(): ?boolean {
```

Comment:
:+1: for a new method.

How about naming it `isSkipped(): boolean` to match the `isScalar()`, `isGenerated` pattern? Also note that the return type can be a non-null `boolean`.